src/client.ts

```
@@ -1368,6 +1393,132 @@ export class BikeTagClient extends EventEmitter {
+              client,
+              DataTypes.game
+            ),
+          })
+          break
+        case AvailableApis.imgur:
+          // eslint-disable-next-line no-case-declarations
+          const getTags = this.getPassthroughApiMethod(
+            api.getTags,
+            client,
+            DataTypes.tag
+          )
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)